src/comps/ConnectWallet.tsx

```diff
@@ -44,10 +44,8 @@ const ConnectWallet = ({ onClose }: ConnectWalletProps) => {
   const { WALLET_NETWORK } = useAtomValue(bridgeConfigAtom);
   const handleSelectWallet = async (wallet: WalletProvider) => {
     try {
-      let addresses = {
-        payment: "",
-        taproot: "",
-      };
+      let addresses: Awaited<ReturnType<typeof getAddressesLeather>> | null =
+        null;
```

Comment:
Will this work with Xverse?

Reply:
Yes this is an abstract interface I just didn't want to export the abstract type but it's worth fixing and using the abstract interface